packages/web/src/components/result/ReactiveList.js

```diff
@@ -496,8 +491,8 @@ ReactiveList.defaultProps = {
 const mapStateToProps = (state, props) => ({
 	defaultPage: (
 		state.selectedValues[`${props.componentId}-page`]
-		&& state.selectedValues[`${props.componentId}-page`].value - 1
-	),
+		&& state.selectedValues[`${props.componentId}-page`].value
+	) || 0,
```

Comment:
You were right, we need an 'invalid' value here to determine the cases where defaultPage is not set. We can use `-1` here, to indicate the same. What are your thoughts?

Reply:
cc @lopper